src/content/governance/index.md

```diff
@@ -0,0 +1,159 @@
+
+4. **EIP Included in Network Upgrade**: assuming the EIP is approved, tested and implemented, it gets scheduled as part of a network upgrade. Given the high coordination costs of network upgrades (everyone needs to upgrade simultaneously), EIPs are generally bundled together in upgrades.
+
+5. **Network Upgrade Activated**: after the network upgrade is activated, the EIP will be live on the Ethereum network. _Note: network upgrades are usually activated on testnets before being activated on the Ethereum Mainnet._
+
+This flow, while very simplified, gives an overview of the significant stages for a protocol change to be activated on Ethereum. Now, let's look at the informal factors at play during this process.
+
+## Informal Process {#informal-process}
+
+### Understanding Prior Work {#prior-work}
+
+EIP Champions should familiarise themselves with prior work and proposals before creating an EIP which can be seriously considered for deployment on the Ethereum Mainnet. This way, the EIP hopefully brings something new which hasn't been rejected before. The three main places to research this are the [EIP repository](https://github.com/ethereum/eips), [Ethereum Magicians](https://www.ethereum-magicians.org/) and [ethresear.ch](https://www.ethresear.ch/).
+
+### Working Groups {#working-groups}
+
+An initial EIP is unlikely to be implemented on the Ethereum Mainnet without edits or changes. Generally, EIP Champions will work with a subset of Protocol Developers to specify, implement, test, iterate, and finalize their proposal. Historically, these working groups have required several months (and sometimes years!) of work. Similarly, EIP Champions for such changes should involve relevant Application/Tooling Developers early in their efforts to gather end-user feedback and mitigate any deployment risks.
+
+### Community Consensus {#community-consensus}
+
+Unfortunately, there is no clear playbook on how to handle contentious proposals. Since they have no way to force people to adopt network upgrades, Protocol Developers will generally avoid including EIPs that are contentious amongst various stakeholder groups.
+
+As the champion of a contentious EIP, you should try and build consensus, address objections and document community support for your proposal. Given the size and diversity of the Ethereum community, there isn't a single metric (e.g. a coin vote) that can be used, and EIP Champions are expected to adapt to the circumstances of their proposal.
+
+Beyond the security of the Ethereum network, significant weight is usually placed on what Application/Tooling Developers and Application Users value, given that their using and developing on Ethereum is what makes the ecosystem attractive for other stakeholders.
+
+<Divider />
+
```

Comment:
Same comment as above regarding "community consensus".  Suggesting to people that this is a requirement and/or will help an EIP (if achieved) or harm an EIP (if it is failed to be achieved) just results in a bunch of people trying to be very loud about their feelings about a particular change.

I think we should strongly encourage people to solicit feedback from different stakeholders, but at the same time I think we should make it clear to readers that they don't need to go out and *sell* their change to the community, and they don't need to go out and convince everyone to hate a particular EIP.

I think the last sentence here would be good as a comment on what core developers have historically favored, which is a section here that I think is missing.  As part of the informal process, you *do* need to convince someone to implement your change in each of the major Ethereum clients.  Part of this process usually means convincing core developers that a particular change is valuable and showing how it helps end-users or solves a security issue can assist in that.

Reply:
Yeah, this is a murky section to write. Here is a changed one. LMK what you think:

```suggestion
### Community Consensus {#community-consensus}

While some EIPs are straightforward technical improvements with minimal nuance, some are more complex and inherently tradeoffs which will affect different stakeholders in different ways. This means some EIPs end up being more contentious within the community than others. 

Unfortunately, there is no clear playbook on how to handle contentious proposals. Since they have no way to force people to adopt network upgrades, Protocol Developers will generally avoid including EIPs that are contentious amongst various stakeholder groups.

As the champion of a contentious EIP, you should try and address objections to build consensus around your EIP. Given the size and diversity of the Ethereum community, there isn't a single metric (e.g. a coin vote) that can be used to gauge community consensus, and EIP Champions are expected to adapt to the circumstances of their proposal. This being said, EIP Champions are expected to solicit feedback from all relevant stakeholders.

Beyond the security of the Ethereum network, significant weight has historically been placed by Protocol Developers on what Application/Tooling Developers and Application Users value, given that their using and developing on Ethereum is what makes the ecosystem attractive for other stakeholders. Additionally, EIPs need to be implemented across all client implementations. Part of this process usually means convincing Protocol Developers that a particular change is valuable and showing how it helps end-users or solves a security issue can assist in that.

<Divider />

```